app/src/main/res/layout/create_event_fragment.xml

```diff
@@ -108,32 +108,46 @@
+                        android:weightSum="1"
+                        android:layout_marginTop="10dp"
+                        android:orientation="horizontal">
 
-                        <AutoCompleteTextView
-                            android:id="@+id/cr_event_address"
-                            android:layout_width="match_parent"
+                        <com.google.android.material.textfield.TextInputLayout
+                            android:layout_width="0dp"
                             android:layout_height="match_parent"
-                            android:backgroundTint="#919090"
-                            android:drawableLeft="@drawable/ic_location_on_black_24dp"
-                            android:drawablePadding="5dp"
-                            android:hint="Address"
-                            android:inputType="textPostalAddress"
-                            android:maxLines="1"
-                            android:paddingLeft="5dp"
-                            android:singleLine="true"
-                            android:textColor="#3C3C3C"
-                            android:textColorHint="#919090"
-                            android:textSize="@dimen/mainScreen_text_size" />
-
+                            android:layout_weight="0.8"
+                            app:hintEnabled="false">
 
-                    </com.google.android.material.textfield.TextInputLayout>
+                            <AutoCompleteTextView
+                                android:id="@+id/cr_location_search"
+                                android:layout_width="match_parent"
+                                android:layout_height="match_parent"
+                                android:backgroundTint="#919090"
+                                android:drawablePadding="5dp"
+                                android:hint="Address"
+                                android:inputType="textPostalAddress"
+                                android:maxLines="1"
+                                android:enabled="false"
+                                android:paddingLeft="5dp"
+                                android:singleLine="true"
+                                android:textColor="#3C3C3C"
+                                android:textColorHint="#919090"
+                                android:textSize="@dimen/mainScreen_text_size"
+                                tools:ignore="HardcodedText" />
 
+                        </com.google.android.material.textfield.TextInputLayout>
+                        <ImageButton
+                            android:id="@+id/img_search_location"
+                            android:layout_width="0dp"
+                            android:layout_height="50dp"
+                            android:layout_weight="0.2"
+                            android:textSize="@dimen/mainScreen_text_size"
+                            android:src="@drawable/ic_location_on_black_24dp"/>
+                    </LinearLayout>
```

Comment:
@HammyHammer this change will affect you because you're working on this file too. Make sure when this PR is merged that you pull from `dev_branch` and update your version with this. You need to include this when merging the conflicts.